packages/host/app/components/operator-mode.gts

```diff
@@ -282,83 +303,101 @@ export default class OperatorMode extends Component<Signature> {
 
       <CardCatalogModal />
 
-      <div class='operator-mode-card-stack'>
-        {{! z-index and offset calculation in the OperatorModeOverlays operates under assumption that it is nested under element with class operator-mode-card-stack }}
-        <OperatorModeOverlays
-          @renderedLinksToCards={{this.renderedLinksToCards}}
-          @addToStack={{this.addToStack}}
-        />
-
-        {{#each this.stack as |item i|}}
-          <div
-            class='operator-mode-card-stack__card'
-            data-test-stack-card-index={{i}}
-            data-test-stack-card={{item.card.id}}
-            style={{this.styleForStackedCard this.stack i}}
-          >
+      {{#if (eq this.stack.length 0)}}
+        <div class='operator-mode__add-card'>
+          <p class='operator-mode__add-card__title'>Add a card to get started</p>
+          {{!-- Cannot find an svg icon with plus in the box 
+          that we can fill the color of the plus and the box. --}}
+          <button class='operator-mode__add-card__button' {{on 'click' this.add}}>
+            <IconButton
```

Comment:
I had to wrap the icon button within another button since I couldn't find an SVG icon for the plus box that allowed me to independently fill the color of the plus and the box.

Reply:
Updated https://github.com/cardstack/boxel/commit/fb3a2bdfb2e574dbaca955f0a8e6221d0a352f44